Content.Server/SimpleStation14/BloodstreamFiller/Components/BloodstreamFillerComponent.cs

```diff
@@ -0,0 +1,167 @@
+using Content.Shared.Damage;
+using Content.Shared.Damage.Prototypes;
+using Robust.Shared.Audio;
+
+namespace Content.Server.SimpleStation14.BloodstreamFiller.Components;
+
+[RegisterComponent]
+public sealed class BloodstreamFillerComponent : Component
+{
+    /// <summary>
+    ///     The duration of the fill DoAfter, in seconds.
+    /// </summary>
+    [DataField("fillTime"), ViewVariables(VVAccess.ReadWrite)]
+    public float FillTime = 1.0f;
+
+    /// <summary>
+    ///     The multiplier for the DoAfter time when attempting to fill yourself.
+    /// </summary>
+    [DataField("selfFillMutli"), ViewVariables(VVAccess.ReadWrite)]
+    public float SelfFillMutli = 3.5f;
+
+    /// <summary>
+    /// The name of the volume to refill.
+    /// </summary>
+    /// <remarks>
+    /// Should match the <see cref="SolutionContainerComponent"/> name or otherwise.
+    /// </remarks>
```

Comment:
```suggestion
    /// <summary>
    ///     The name of the volume to refill.
    /// </summary>
    /// <remarks>
    ///     Should match the <see cref="SolutionContainerComponent"/> name or otherwise.
    /// </remarks>
```